playbooks/common/openshift-cluster/redeploy-certificates/ca.yml

```diff
@@ -294,7 +294,7 @@
       client_path: "{{ openshift.common.config_base }}/master/admin.kubeconfig"
       ca_path: "{{ openshift.common.config_base }}/master/ca-bundle.crt"
   - name: Lookup default group for ansible_ssh_user
-    command: "/usr/bin/id -g {{ ansible_ssh_user }}"
+    command: "/usr/bin/id -g {{ ansible_ssh_user | quote }}"
```

Comment:
Could you use the `real_group_id` fact from: https://github.com/ansible/ansible/tree/devel/lib/ansible/module_utils#L546-L558, it should be available as `ansible_real_group_id`